src/base/index.vue

```diff
@@ -1340,6 +1346,12 @@ export default {
         "Your section has been removed, save your page to display this change to your visitors"
       );
     },
+    copyAnchor(anchor) {
+      // let copyText = document.getElementById("myInput");
+      // copyText.select();
+      // copyText.setSelectionRange(0, 99999);
+      navigator.clipboard.writeText(anchor);
```

Comment:
given this is just one function, couldn't you simply call it in the @click event ?